README.md

```diff
@@ -51,26 +49,26 @@ The [OpenStack charm guide](https://docs.openstack.org/charm-guide)provides guid
 
 The [OpenStack charms deployment guide](https://docs.openstack.org/project-deploy-guide/charm-deployment-guide) demonstrates how to deploy and configure charms manually to build an OpenStack cloud.
 
-# Project and community
+## Project and community
 
 We warmly welcome community contributions, suggestions, fixes, and constructive feedback.
-If you find any errors or have suggestions for improvements, please report a bug on launchpad on the specific component (For example, for ceph dashboard, use [this link](https://bugs.launchpad.net/charm-ceph-dashboard).
+If you find any errors or have suggestions for improvements, please [report a bug on launchpad](https://bugs.launchpad.net/) on the specific component (For example, [report a ceph-dashboard charm bug](https://bugs.launchpad.net/charm-ceph-dashboard) in the Ceph Dashboard charm project on Launchpad..
```

Comment:
I believe `(for example...` needs to be lowercase, also we're missing a closing paren